TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -87,7 +111,7 @@ const BenchmarkMetricRegex = {
         metrics: {
             "Score":{
                 //Example: ArrayListSortComparatorBench score: 2757481.000000 (2.757M 1483.0%)
-                regex: /ArrayListSort[\s\S\n]*? score: ?(\d*\.?\d*)/,
+                regex: /[\s\S\n]*? score: ?(\d*\.?\d*)/,
```

Comment:
If we use `ArrayListSort`, then we won't be able to parse other tests from BumbleBench such as the ones for string, lambda and other components.

```
  <Bumble Bench Test Name> score: 1.1859834E7 (11.86M 1628.9%)
```

Reply:
So do you agree that I remove the `ArrayListSort`?

Reply:
Since this is a cleanup task and not a feature request, I would like to make sure this task is focusing on its goal. In this way, it is easier for us to do cleanup and testing the changes.

If the original code cannot parse other BumbleBench, please create a separate PR to update it.
Also, for the example mentioned above, removing `ArrayListSort` does not help much. The code will return `1.1859834` for `1.1859834E7`. `E7` will not be captured.

Reply:
Sounds good! Only clean up related changes.